examples/aws/tidb-cluster.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: pingcap.com/v1alpha1
```

Comment:
This is used in the official EKS deployment doc, so we have to provide a yaml file that can be used in production env, please include the necessary configurations in the https://github.com/pingcap/tidb-operator/blob/master/deploy/aws/manifests/db.yaml.example

Reply:
done